CLAUDE.md

```
@@ -391,6 +418,53 @@ Props: title, description, date, imageUrl, href
+| `style` | 코드 포맷팅, 스타일 변경 (기능 변경 없음) |
+| `refactor` | 코드 리팩토링 (기능 변경 없음) |
+| `ci` | CI/CD, 빌드, 설정 파일 변경 |
+| `chore` | 패키지 설치, 기타 잡무 |
+| `test` | 테스트 코드 추가/수정 |
+
+### 커밋 시 지켜야하는 규칙
+- 커밋 메세지에서 클로드를 협업자로 추가하지 않는다. (Co-Authored-By: Claude ~ 와 같은 내용을 추가하지 않는다. )
+
+
+### 브랜치명 컨벤션
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)